public/css/sequenceserver.css

```diff
@@ -507,3 +421,8 @@ td.nowrap-ellipsis {
   display: flex;
   justify-content: space-between;
 }
+
+/* override tailwind class  */
+.svg-container.collapse {
```

Comment:
Would it be better to change this CSS class name in JSX instead of fighting tailwind?

Reply:
Yeah, I'll update

Reply:
Updated.